docs/SceneWalkthrough/AchievementsWalkthrough.md

```diff
@@ -0,0 +1,11 @@
+## **Achievements Demo**
+This demo showcases the Achievements function. This is done through 2 test achievements, one logging logins and the other logging an arbitrary stat. Each Achievement has a unlocked and locked display name and description, as well as the ability to hide it and the relevant tracked data for it.
```

Comment:
Line 2 : logging logins confused a bit at first glance, maybe tracking login count could be better